spark/dl/src/main/scala/com/intel/analytics/bigdl/models/utils/ModelBroadcast.scala

```diff
@@ -80,11 +85,18 @@ class ModelBroadcast[T: ClassTag](implicit ev: TensorNumeric[T]) extends Seriali
       if (parameters._1(i) != null) {
         parameters._1(i).set()
```

Comment:
QuanTensor in clone module should not be released, while set() will cal release.

On the other hand, cloneModule uses serialization, and desc is transient and will be 0 in cloned module. Therefore the release() call will not work either.

Do we really need to release QuanTensor?

Reply:
We don't call it in code except the test cases. In the example models, the memory (QuantTensor) will be released at the end of the process.

Reply:
Another case is, if we set a tensor from another tensor, we should release the old tensor's memory first.